nomad/poc-2/init.sh

```diff
@@ -0,0 +1,27 @@
+#/usr/bin/env bash
+set -xeu
+
+pwd
+ls -lha
+
+export JOB_STATE_DIRECTORY="${STATE_DIRECTORY}/${NOMAD_JOB_ID}"
+
+mkdir -p "${JOB_STATE_DIRECTORY}"
+pushd "${JOB_STATE_DIRECTORY}"
```

Comment:
`trap` and `popd` to clean up or switch to `cd`?

Reply:
the aim here was to have a persistent directory specifically to keep the target directory. i plan to write cleanup logic once this comes out of POC territory.

Reply:
i added a comment to explain this